api/cookies.go

```diff
@@ -41,6 +41,18 @@ func NewSessionCookieForConsole(token string) http.Cookie {
 	}
 }
 
+// NewIDPSessionCookie creates a cookie for a refresh token
+func NewIDPSessionCookie(token string) http.Cookie {
+	return http.Cookie{
+		Path:     "/",
```

Comment:
I would suggest to change the cookie path to `/api/v1/`, so it will only be sent for API requests. We also may consider to move `/login` and `/logout` to `/auth/login` and `/auth/logout` and scope the cookie to `/api/v1/auth/`. Then the cookie will only be sent for requests that deal with logging in/out.

We also may scope the normal console cookie to `/api/v1/` to reduce cookie traffic.

Reply:
Changed cookies path to `/api/v1` and did some testing, doesn't seem to break anything, done!.

About change the login & logout API paths to `/api/v1/auth/`, it's a bit out of scope of this issue and can potentially be a breaking change, It would like to not address that in this PR to do not hold the IDP logout.

Reply:
I didn't look into the console token, but we may need to call IDP (using the refresh-token) when the console token expires anyway. In that case, we do need to send the IDP token with every request anyway.